src/sync/installation.ts

```diff
@@ -413,38 +413,23 @@ async function doProcessInstallation(app, queues, job, installationId: number, j
 			return;
 		}
 
-		if (await booleanFlag(BooleanFlags.CONTINUE_SYNC_ON_ERROR, false, jiraHost)) {
```

Comment:
The feature flag is currently enabled on all environments, so we can just remove the `else` branch.